cli/src/tests/helpers/dirs.rs

```diff
@@ -3,7 +3,7 @@ lazy_static! {
     static ref FIXTURES_DIR: PathBuf = ROOT_DIR.join("test").join("fixtures");
     static ref HEADER_DIR: PathBuf = ROOT_DIR.join("lib").join("include");
     static ref GRAMMARS_DIR: PathBuf = ROOT_DIR.join("test").join("fixtures").join("grammars");
-    static ref SCRATCH_DIR: PathBuf = {
+    pub static ref SCRATCH_DIR: PathBuf = {
```

Comment:
is this ok ?

Reply:
maybe `pub(crate)` to avoid accidental exports in the future?

Reply:
Instead - maybe we can use python and set the regex to check for `#!` and then python somewhere after.

Reply:
I've pushed a better change for this, if you're good w/ that do you mind squashing the commits down to 1? Or I can if you're wary about doing that.

Reply:
Yeah it's fine I leave that to you  I'm used to commiting random stuff and the maintainers squash and merge